ProjectSolenodon/Assets/Scripts/LevelData.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.IO;
+
+/// <summary>
+/// The LevelData class contains holds data that the LevelManager uses to create the scene for a chosen level.
+/// Members here are pieces of data that variate from level to level.
+/// </summary>
+[Serializable]
+public class LevelData
+{
+    public int numRegisterCards;
+    public int numStackCards;
+    public int numQueueCards;
+    public int numHeapCards;

```

Comment:
Declare private and provide getters.

I'd argue against adding setters.

Reply:
Alternatively, can this also be done via start() for a level?